test/Microsoft.AspNetCore.ResponseCaching.Tests/TestUtils.cs

```diff
@@ -30,18 +31,29 @@ static TestUtils()
             StreamUtilities.BodySegmentSize = 10;
         }
 
-        internal static RequestDelegate TestRequestDelegate = async (context) =>
+        internal static RequestDelegate TestRequestDelegate = async context =>
         {
-            var uniqueId = Guid.NewGuid().ToString();
             var headers = context.Response.GetTypedHeaders();
-            headers.CacheControl = new CacheControlHeaderValue()
+
+            string expires = context.Request.Query["Expires"];
+            if (!string.IsNullOrEmpty(expires))
+            {
+                headers.Expires = DateTimeOffset.Parse(expires);
```

Comment:
This looks like a lot of hassle to set in the test, how about just passing this in seconds as an `int`?